src/gui/guiFormSpecMenu.cpp

```diff
@@ -655,7 +655,7 @@ void GUIFormSpecMenu::parseCheckbox(parserData* data, const std::string &element
 		auto style = getDefaultStyleForElement("checkbox", name);
 		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, false));
 
-		if (spec.fname == data->focused_fieldname) {
+		if (spec.fname == m_focused_element) {
```

Comment:
Worth noting that I'm moving things _into_ RenderData in my formspec refactor, not out

Reply:
I added it here because the others wanted to use `setFocus`, which can't access `parserData`; trying to move stuff around probably wouldn't help the refactor much.